frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -1,41 +1,34 @@
 import * as _ from 'lodash';
-import { k8sGet, K8sResourceKind } from '@console/internal/module/k8s';
+import { Dispatch } from 'react-redux';
+import { k8sGet } from '@console/internal/module/k8s';
 import { ClusterServiceVersionModel, SubscriptionModel } from '@console/operator-lifecycle-manager';
-import { handleError, setFlag } from '@console/internal/actions/features';
+import { setFlag } from '@console/internal/actions/features';
 import { FeatureDetector } from '@console/plugin-sdk';
+import { getAnnotations } from '@console/shared/src/selectors/common';
 import { OCSServiceModel } from './models';
 import {
   OCS_INDEPENDENT_CR_NAME,
   CEPH_STORAGE_NAMESPACE,
   OCS_SUPPORT_ANNOTATION,
+  OCS_CONVERGED_CR_NAME,
 } from './constants';
-import { getAnnotations } from '@console/shared/src/selectors/common';
 
 export const OCS_INDEPENDENT_FLAG = 'OCS_INDEPENDENT';
-
-const isIndependent = (data: K8sResourceKind): boolean =>
-  data.spec?.externalStorage?.enable ?? false;
-
-export const detectIndependentMode: FeatureDetector = (dispatch) =>
-  k8sGet(OCSServiceModel, OCS_INDEPENDENT_CR_NAME, CEPH_STORAGE_NAMESPACE).then(
-    (obj: K8sResourceKind) => dispatch(setFlag(OCS_INDEPENDENT_FLAG, isIndependent(obj))),
-    (err) => {
-      err?.response?.status === 404
-        ? dispatch(setFlag(OCS_INDEPENDENT_FLAG, false))
-        : handleError(err, OCS_INDEPENDENT_FLAG, dispatch, detectIndependentMode);
-    },
-  );
+export const OCS_CONVERGED_FLAG = 'OCS_CONVERGED';
+export const OCS_FLAG = 'OCS';
```

Comment:
Please add a comment that this flag enables `noobaa` as the name itself is not explanatory.

Reply:
The name is self-explanatory. As OCS encompasses both Noobaa and Ceph.

Reply:
But here this flag is just used to enable "Object Service Dashboard" and not "persistent dashboard"

Reply:
Please refer to the other plugin file.

Reply:
All the flags which have defined across the plugins are self explanatory and hence, they don't need any comment. I asked you to add a comment here as  OCS encompasses both Noobaa and Ceph, but `OCS_FLAG` is used to enable only Object Service Dashboard(creates confusion, at least I was).  Else rename the flag to `OBJECT_SERVICE_DASHBOARD_FLAG` which is more precise.

Reply:
I don't see this conversation as resolved. Can you explain the reason for resolving this?

Reply:
By mistake I deleted the earlier comment, but I see that you have added the comment. Thanks!